static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -95,6 +96,7 @@ function InviteMembersModal({
               setRole={setRole}
               setTeams={setTeams}
               willInvite={willInvite}
+              error={error}
```

Comment:
It's a bit dependent on the modal, but it'd be nice if we didn't have to do prop drilling here -- something tells me it's needed given other components tho.

Reply:
yeah it would've been nice to just update a context or state and have it done automatically